libtiny_tui/src/messaging.rs

```diff
@@ -284,22 +284,30 @@ impl MessagingUI {
         self.msg_area.flush_line();
     }
 
+    fn _add_client_msg(&mut self, msg: &str) {
+        // Replace tabs with 4 spaces
+        let msg = msg.replace("\t", "    ");
+        let lines = msg.split('\n');
+        for line in lines {
+            self.msg_area.add_text(line);
+            self.msg_area.flush_line();
+        }
+    }
```

Comment:
Use 4 spaces instead of `\t` and you won't need this.

`\t` is really tricky to use with termbox so it's better if we use spaces instead.